device_calendar/lib/src/common/calendar_enums.dart

```diff
@@ -134,4 +140,12 @@ extension IntExtensions on int {
   DayOfWeek get getDayOfWeekEnumValue => _getDayOfWeekEnumValue(this);
   MonthOfYear get getMonthOfYearEnumValue => _getMonthOfYearEnumValue(this);
   WeekNumber get getWeekNumberEnumValue => _getWeekNumberEnumValue(this);
+}
+
+extension ObjectExtensions on Object {
```

Comment:
Can we make this explicit to the type of enum we want to convert to string (eg https://stackoverflow.com/questions/38908285/add-methods-or-values-to-enum-in-dart)